### ISSUE TITLE
Cant Install Mobile on Few Androids

### ISSUE THREAD

**Issue by Diego-OT** [OTCv8/otclientv8#208]
Since some date in February, something has changed, a lot of people haven't been able to install the Mobile client anymore.
The old one they can install normal, but it doesn't work because of the new features.

**Comment by zafrim** [comment]
Exactly, after i update to latest version of otcv8.  A LOT people claim they lost compatibility because their cell phone is not 64bit, The only way is to use outdated versions of mobile
